.github/workflows/continuous-integration-workflow.yml

```diff
@@ -1,11 +1,40 @@
 name: Continous Integration Tests
-on:
-  pull_request:
-    branches:
-      - master
+on: [push, pull_request]
 jobs:
-  test:
-    name: ODS resources setup and provisioning tests
+  webhook-proxy:
+    name: Webhook Proxy tests
+    runs-on: ubuntu-18.04
+    steps:
+      -
+        name: Checkout repository
+        uses: actions/checkout@v2.0.0
+        with:
+          fetch-depth: 0
```

Comment:
You can use default fetch-depth (or fetch-depth 1) if you do not need the whole git history. It is faster

Reply:
👍